src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            Driver driver = null;
+            if (resultSet.next()) {
+                driver = createDriverFromDB(resultSet);
+            }
+            return Optional.ofNullable(driver);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver by id: " + id, e);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        List<Driver> allDrivers = new ArrayList<>();
+        String getAllQuery = "SELECT * FROM drivers WHERE is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllDriversStatement =
+                        connection.prepareStatement(getAllQuery)) {
+            ResultSet resultSet = getAllDriversStatement.executeQuery(getAllQuery);
+            while (resultSet.next()) {
+                allDrivers.add(createDriverFromDB(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all drivers from db", e);
+        }
+        return allDrivers;
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateQuery = "UPDATE drivers SET name = ?, licenseNumber = ?"
+                + " WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateStatement = connection.prepareStatement(updateQuery)) {
+            updateStatement.setString(1, driver.getName());
+            updateStatement.setString(2, driver.getLicenseNumber());
+            updateStatement.setObject(3, driver.getId());
+            updateStatement.executeUpdate();
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update a driver " + driver, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE drivers SET is_deleted = TRUE WHERE id =?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteStatement = connection
+                        .prepareStatement(deleteQuery)) {
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete a driver with id: " + id, e);
+        }
+    }
+
+    private static Driver createDriverFromDB(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("id", Long.class);
+        String name = resultSet.getString("name");
+        String licenseNumber = resultSet.getString("licenseNumber");

```

Comment:
```suggestion
        String licenseNumber = resultSet.getString("license_number");

```

Reply:
my bad(, It was an auto-generated method with comb ctrl + alt + m